File: C/Iniciante/1074/1074.c
#include <stdio.h>

int main()
{
    int N, X, cont;
    scanf("%d", &N);
    for (cont = 1; cont <= N; cont++)
    {
        scanf("%d", &X);
        if (X == 0)
            printf("NULL\n");
        else if (X <= 0 && X % 2 == 0)
            printf("EVEN NEGATIVE\n");
        else if (X <= 0 && X % 2 == -1)
            printf("ODD NEGATIVE\n");
        else if (X >= 0 && X % 2 == 0)
            printf("EVEN POSITIVE\n");
        else if (X >= 0 && X % 2 == 1)
            printf("ODD POSITIVE\n");
    }
    return 0;
}

File: Rust/Iniciante/1074.rs
use std::io;
use std::error::Error;

fn main() -> Result<(), Box<dyn Error>> {
	let stdin = io::stdin();
	let mut buffer = String::new();

	stdin.read_line(&mut buffer)?;

	let n: i64 = buffer.trim().parse()?;

	for _ in 0..n {
		buffer.clear();
		stdin.read_line(&mut buffer)?;

		let m: i64 = buffer.trim().parse()?;

		if m == 0 {
			println!("NULL");
		} else if m % 2 == 0 {
			if m > 0 {
				println!("EVEN POSITIVE");
			} else {
				println!("EVEN NEGATIVE");
			}
		} else {
			if m > 0 {
				println!("ODD POSITIVE");
			} else {
				println!("ODD NEGATIVE");
			}
		}
	}

	Ok(())
}